src/use_cases_execution/birthday/fetch_birthdays_from_google.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require 'json'
+require 'bas/shared_storage/postgres'
+require 'bas/shared_storage/default'
+require_relative 'config'
+
+module Routes
+  # Routes::Birthdays handles incoming birthday data from Google Workspace
+  #
+  # POST /birthday - Accepts JSON payload with birthday data
+  # Expected format: { "birthdays": [array of birthday objects] }
+  # Returns: 200 with success message or 400/500 with error details
+  class Birthdays < Sinatra::Base
+    write_options = {
+      connection: Config::CONNECTION,
+      db_table: 'birthday',
+      tag: 'FetchBirthdaysFromGoogle'
```

Comment:
What do you think about using a more descriptive and appropriate tag? 
```suggestion
      tag: 'BirthdaysFromGoogle'
```